jupyterhub_singleuser_profiles/images.py

```diff
@@ -27,6 +27,16 @@ def get_default(self):
 
         return ''
 
+    def get_info(self, image_name):
+        imagestream_list = self.openshift.get_imagestreams(IMAGE_LABEL+'=true')
+
+        if len(imagestream_list.items) == 0:
+            return None
+        else:
+            for i in imagestream_list.items:
+                if i.metadata.name == image_name:
+                    return dict(i.metadata.annotations)
```

Comment:
This should not just return annotations as they are..we need to create a schema for this, not return all annotations